tests/Unit/FramesMonitor/_runShowFramesProcess.test.js

```diff
@@ -6,24 +6,26 @@ const {assert}   = require('chai');
 
 const {config, url} = require('./Helpers');
 
-function getSpawnArguments(url, timeoutInSec, errorLevel) {
+function getSpawnArguments(url, timeoutInMs, errorLevel) {
     return [
         '-hide_banner',
         '-v',
         errorLevel,
         '-fflags',
         'nobuffer',
+        '-rw_timeout',
+        timeoutInMs * 1000,
```

Comment:
Seems like you don't need to multiply by 1000

Reply:
It`s all right, because `getSpawnArguments()` receive value in milliseconds but FFprobe expects microseconds.